sdk/python/kfp/cli/pipeline.py

```diff
@@ -57,10 +63,18 @@ def upload(ctx, pipeline_name, package_file):
 )
 @click.argument("package-file")
 @click.pass_context
-def upload_version(ctx, package_file, pipeline_version, pipeline_id):
+def upload_version(ctx, package_file, pipeline_version, pipeline_id=None, pipeline_name=None):
     """Upload a version of the KFP pipeline"""
     client = ctx.obj["client"]
-
+    if pipeline_id==None and pipeline_name==None: #[TODO] do this using click instead... 
+        logging.Warning("Need to suppy 'pipeline-name' or 'pipeline-id'")
```

Comment:
This probably needs to be an error.

Reply:
Will update with value error on both places.